system/Helpers/text_helper.php

```diff
@@ -880,4 +880,29 @@ function excerpt(string $text, string $phrase = null, int $radius = 100, string
 	}
 
 	//--------------------------------------------------------------------
+	if (! function_exists('replace_string'))
+{
+	/**
+	 * Remove String.
+	 *
+	 * Allows to removed specific keyword or fields from string
+	 *
+	 * @param string  $string Given String
+	 * @param array  $values    Given value that will be replace
+	 * @param integer $replace   Matches any of the patterns are replaced with the replacement string.
+	 *
+	 * @return string
+	 *
+	 */
+	function replace_string(string $string, array $values, string $replace = ''): string
+	{
+		foreach ($values as $value)
+		{
+			$string = preg_replace("~{$value}~s", $replace, $string);
+		}
+		return $string;
+	}
+
```

Comment:
missing close `}`, also, again, please provide unit tests and follow user guide, do composer update and it will automatically fix it on next commit.